azure-pipelines.yml

```diff
@@ -0,0 +1,58 @@
+name: HMCTS Shared Services AKS Azure Infrastructure Component Deployment Pipeline
+trigger:
+  - none
+
+parameters:
+  - name: components
+    displayName: Component to Run
+    type: object
+    default:
+      - frontdoor
+      - appgateway
+  - name: location
+    displayName: Location
+    type: string
+    default: 'UK South'
+    values:
+      - 'UK South'
+      - 'UK West'
+  - name: env
```

Comment:
I would prefer we map to the same convention as in azure-platform-terraform where there's a list of objects that includes all envs and you don't need to run it individually against each env.

with this current approach we don't really know what env something has been run against and it's really easy for environment drift

Reply:
modified parameters in the yaml file